app/Models/BeatmapsetEvent.php

```diff
@@ -92,6 +92,7 @@ public static function search($rawParams = [])
 
         $query = static::limit($params['limit'])->offset($pagination['offset']);
         $searchByUser = present($rawParams['user'] ?? null);
+        $searchByBeatmapset = present($rawParams['beatmapset'] ?? null);
```

Comment:
can just be in the `if` condition

Reply:
sure, was trying to be consistent but i see they're that `$searchByUser` is used in more places than one